spec/controllers/meetings_controller_spec.rb

```diff
@@ -39,7 +39,7 @@
       ).and_call_original
 
       expect_any_instance_of(MeetingStarter).to receive(:call)
-      expect_any_instance_of(BigBlueButtonApi).to receive(:join_meeting).with(room:, name: user.name, role: 'Moderator')
+      expect_any_instance_of(BigBlueButtonApi).to receive(:join_meeting).with(room:, name: user.name, avatar_url: nil, role: 'Moderator')
```

Comment:
Changing the interface can help stabilize the interface and remove the excessive need to update mocks and stubs when updating the method implementation,